src/InvariantTheory/InvariantTheory.jl

```diff
@@ -6,3 +6,6 @@ include("primary_invariants.jl")
 include("secondary_invariants.jl")
 include("fundamental_invariants.jl")
 include("affine_algebra.jl")
+
+# postponed inclusion from src/Modules/Modules.jl due to dependency
+include("../Modules/Iterators.jl") 
```

Comment:
This is not really about this PR, but it seems to me like we should start thinking about putting all types (`(mutable) struct`s) in a handful of 'central' places or at least make sure that they can be included at first.

Although in this case we could also just move the `all_monomials` for rings to the `Rings/` directory where it arguably belongs.

Reply:
Yes, this discussion has to take place at some point. But not here, eventually.